docs/index.rst

```diff
@@ -1,14 +1,162 @@
-Welcome to Betelgeuse's documentation!
-======================================
+Betelgeuse documentation
+========================
 
-Betelgeuse reads standard Python test cases and offers tools to interact with
-Polarion. Possible interactions are:
+.. contents:: Topics
 
-* Automatic creation and update of Requirements and Test Cases from a Python
+.. _what_is_betelgeuse:
+
+What is Betelgeuse?
+```````````````````
+
+Betelgeuse is a python program that reads standard Python test cases and offers
+tools to interact with Polarion. Possible interactions are:
+
+* Automatic creation/updation of Requirements and Test Cases from a Python
   project code base.
 * Creation of Test Runs based on a jUnit XML file.
 
-Documentation contents:
+Betelgeuse uses `testimony <https://pypi.python.org/pypi/testimony>`_ and
+pylarion projects to parse test cases and communicate with polarion
+respectively.
+
+.. _prerequisites:
+
+Prerequisites
+`````````````
+Login to polarion in a browser to check if the login user has permissions to do
+the following:
+
+* add/update test case
+* add/update requirements
+* create test runs
+
+.. _quick_start:
+
+Quick Start
+```````````
+1. betelgeuse uses pylarion to interact with polarion. Install pylarion from its
+   source.
+
+   .. note::
+
+     - It may be possible that the latest version of betelgeuse may not work
```

Comment:
I have been told that some teams use original pylarion and other teams using forked version.  So this is a note to the user that he should be mindful of using the right pylarion version.  Also I am refraining from adding too much info on pylarion here as this is an internal project.

In the long run, hopefully our fixes get accepted in original pylarion and we don't need to fork it anymore.  So this is a kind of temporary solution.